app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@Fryguy But we still need rescue code to properly interpret callback action, virtual column or not. Right now rescue part does not really do that and controllers manually create flash messages rather than use those created by the model.